elasticdl/python/ps/embedding_table.py

```diff
@@ -85,6 +85,24 @@ def to_embedding_table_info_pb(self):
         embedding_pb.initializer = str(self.initializer_value)
         return embedding_pb
 
+    def get_table_size(self):
```

Comment:
Is it to get the count of elements in the embedding table? If it is,  we maybe need to add a docstring.

Reply:
`size` represents for the element count in a tensor, in both TensorFlow and numpy. I think that it's clear here. I will add a docstring.